### PR TITLE
Fix integration tests [ECR-3384]

### PR DESCRIPTION
**Pull request by skletsun**
## Overview
Re-enabling disabled integration tests.

---
See: https://jira.bf.local/browse/ECR-3384

### Definition of Done

- [x] There are no TODOs left in the code
- [x] Change is covered by automated [tests](https://github.com/exonum/exonum-java-binding/blob/master/CONTRIBUTING.md#tests)
- [x] The [coding guidelines](https://github.com/exonum/exonum-java-binding/blob/master/CONTRIBUTING.md#the-code-style) are followed
- [x] Public API has Javadoc
- [x] Method preconditions are checked and documented in the Javadoc of the method
- [x] Changelog is updated if needed (in case of notable or breaking changes)
- [x] The continuous integration build passes

### CODE REVIEW

File: exonum-java-binding/core/src/test/java/com/exonum/binding/core/storage/indices/ProofMapIndexProxyIntegrationTest.java [hunk 1]
```diff
@@ -226,15 +226,15 @@ void getRootHash_NonEmptyMap() {
     });
   }
 
-  @Disabled  //FIXME: Tests here and below are disabled until ECR-3320
+  @Disabled("Fails on assertion, disabled until ECR-3416")
```
Comment (dmitry-timofeev) [inline exonum-java-binding/core/src/test/java/com/exonum/binding/core/storage/indices/ProofMapIndexProxyIntegrationTest.java]: Why isn't 3320? (#1032)

**Comment by dmitry-timofeev** [comment]
It doesn't build.

File: exonum-java-binding/core/src/test/java/com/exonum/binding/core/storage/indices/ProofMapIndexProxyIntegrationTest.java (hunk 1 above)
Comment (skletsun) [reply exonum-java-binding/core/src/test/java/com/exonum/binding/core/storage/indices/ProofMapIndexProxyIntegrationTest.java]: Fixed, thanks.

**Comment by vitvakatu** [comment]
Looks like internal Travis error, restarted the job

**Comment by dmitry-timofeev** [comment]
> Looks like internal Travis error, restarted the job

It happens all the time lately 😞 Shall we adopt Github actions?

**Pull request merged**